dev/NavigationView/NavigationView.cpp

```diff
@@ -1594,6 +1734,222 @@ void NavigationView::UpdateVisualStateForDisplayModeGroup(const winrt::Navigatio
     }
 }
 
+void NavigationView::OnNavigationViewItemTapped(const winrt::IInspectable& sender, const winrt::TappedRoutedEventArgs& args)
+{
+    if (auto nvi = sender.try_as<winrt::NavigationViewItem>())
+    {
+        if (IsSettingsItem(nvi))
+        {
+            OnSettingsInvoked();
+        }
+        else
+        {
+            OnNavigationViewItemInvoked(nvi);
+        }
+        nvi.Focus(winrt::FocusState::Pointer);
+    }
+}
+
+void NavigationView::OnNavigationViewItemKeyUp(const winrt::IInspectable& sender, const winrt::KeyRoutedEventArgs& args)
+{
+    // Because ListViewItem eats the events, we only get these keys on KeyUp.
```

Comment:
>// Because ListViewItem eats the events, we only get these keys on KeyUp. [](start = 4, length = 73)

This seems like it might be a good reason not to use ListViewItem. Maybe for a different change.

Reply:
Makes sense, for now it seems to be the only issue with using the ListViewItem, so yeah, for a different change